test/check-machines-hostdevs

```diff
@@ -143,7 +153,7 @@ class TestMachinesHostDevs(VirtualMachinesCase):
                 b.wait_not_present("#vm-subVmTest1-hostdevs-dialog")
 
             def add(self):
-                m.execute("virsh dumpxml subVmTest1 > /tmp/vmxml1")
+                run_admin("virsh dumpxml subVmTest1 > /tmp/vmxml1")
```

Comment:
That looks very clean now, nice! The function name is a bit debatable, but :shrug:

Reply:
I named it just `run` but it's taken already. Any ideas welcome.